### ISSUE TITLE
Update Python/Anaconda installation videos

### ISSUE THREAD

**Issue by maneesha** [datacarpentry/python-ecology-lesson#587]
### What is the problem?

The videos on how to install Anaconda are out of date (using resources from continuum.io)

### Location of problem (optional)

https://datacarpentry.org/python-ecology-lesson/#installing-anaconda

**Comment by quist00** [comment]
Hi, @maneesha

Do you know if one of the other lessons has good videos we can use to replace these, or would it be best just to remove the video guidance and create a help wanted issue to locate new ones?

Alternatively, we could also just consider removing them altogether an rely on whatever guidance is built into the workshop website template. It seems wasteful to keep juggling this given how quickly video can go stale.  What do you think?

**Comment by maneesha** [comment]
Tagging @sstevens2 who originally created these videos to see if she has any other resources to recommend.

My personal preference would be to remove them if we can not realistically keep them up to date.

**Comment by quist00** [comment]
Can you clarify, are you suggesting here https://github.com/carpentries/workshop-template/blob/gh-pages/_includes/swc/setup.html for the additional removals?

**Comment by sstevens2** [comment]
Yes, that is the other location to remove it.  Sorry that wasn't very specific.
